tools.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
+
+package swarmkit
+
+import (
+	_ "github.com/stevvooe/protobuild"
```

Comment:
There was a comment in the makefile about how the `protobuild` tool should be moved into vendor. This does that and allows us to pin to a version (which also made the build easier since we didn't have to try and install this somewhere in the path).